[
    {
        "content": "<p>This week getting new qa error on all LP codes used in examples here: <a href=\"http://build.fhir.org/ig/HL7/US-Core/qa.html\">http://build.fhir.org/ig/HL7/US-Core/qa.html</a></p>\n<p>for example:</p>\n<p>for ...</p>\n<div class=\"codehilite\" data-code-language=\"JSON\"><pre><span></span><code><span class=\"err\">...</span>\n  <span class=\"nt\">\"category\"</span> <span class=\"p\">:</span> <span class=\"p\">[</span>\n    <span class=\"p\">{</span>\n      <span class=\"nt\">\"coding\"</span> <span class=\"p\">:</span> <span class=\"p\">[</span>\n        <span class=\"p\">{</span>\n          <span class=\"nt\">\"system\"</span> <span class=\"p\">:</span> <span class=\"s2\">\"http://loinc.org\"</span><span class=\"p\">,</span>\n          <span class=\"nt\">\"code\"</span> <span class=\"p\">:</span> <span class=\"s2\">\"LP29708-2\"</span><span class=\"p\">,</span>\n          <span class=\"nt\">\"display\"</span> <span class=\"p\">:</span> <span class=\"s2\">\"Cardiology\"</span>\n        <span class=\"p\">}</span>\n      <span class=\"p\">],</span>\n      <span class=\"nt\">\"text\"</span> <span class=\"p\">:</span> <span class=\"s2\">\"Cardiology\"</span>\n    <span class=\"p\">}</span>\n  <span class=\"p\">],</span>\n<span class=\"err\">...</span>\n</code></pre></div>\n<p>Getting these errors and warnings...</p>\n<p>DiagnosticReport/cardiology-report: DiagnosticReport.category[0] (l21/c8)   warning [Unable to determine whether the provided codes are in the value set <a href=\"http://hl7.org/fhir/us/core/ValueSet/us-core-diagnosticreport-category\">http://hl7.org/fhir/us/core/ValueSet/us-core-diagnosticreport-category</a> because the value set or code system is not known to the validator]</p>\n<p>DiagnosticReport/cardiology-report: DiagnosticReport.category[0] (l21/c8)   error   Wrong length index getting LOINC language code (from <a href=\"http://tx.fhir.org/r4\">http://tx.fhir.org/r4</a>)</p>\n<p>DiagnosticReport/cardiology-report: DiagnosticReport.category[0].coding[0] (l23/c14)    error   Wrong length index getting LOINC language code (from <a href=\"http://tx.fhir.org/r4\">http://tx.fhir.org/r4</a>) for 'http://loinc.org#LP29708-2'</p>\n<hr>\n<p><a href=\"https://loinc.org/LP29708-2/\">LP29708-2</a> is a valid LOINC code for the code system '<a href=\"http://loinc.org\">http://loinc.org</a>'</p>",
        "id": 243615261,
        "sender_full_name": "Eric Haas",
        "timestamp": 1624433163
    },
    {
        "content": "<p>have you cleaned out your txCache? The server was returning this error transiently but should not be doing that now</p>",
        "id": 243615452,
        "sender_full_name": "Grahame Grieve",
        "timestamp": 1624433348
    },
    {
        "content": "<p>I cleaned out the tx-cache locally but noticed that the github repro ignores it and the autobuild still gives those same errors.</p>",
        "id": 243617437,
        "sender_full_name": "Eric Haas",
        "timestamp": 1624434739
    },
    {
        "content": "<p>here are more errors I missed from before:  </p>\n<p>for valueset-us-core-diagnosticreport-lab-codes.xml</p>\n<blockquote>\n<p>ValueSet.where(id = 'us-core-diagnosticreport-lab-codes') error   Error from server: Start index out of bounds (-52)</p>\n</blockquote>",
        "id": 243617669,
        "sender_full_name": "Eric Haas",
        "timestamp": 1624434901
    },
    {
        "content": "<p>for valueset-us-core-diagnosticreport-report-and-note-codes.xml</p>\n<blockquote>\n<p>ValueSet.where(id = 'us-core-diagnosticreport-report-and-note-codes')  error   Error from server: Attempt to access invalid LOINC index at 4934540</p>\n</blockquote>",
        "id": 243617689,
        "sender_full_name": "Eric Haas",
        "timestamp": 1624434918
    },
    {
        "content": "<p>adn from valueset-us-core-diagnosticreport-report-and-note-codes.xml<br>\n:</p>\n<blockquote>\n<p>ValueSet.where(id = 'us-core-diagnosticreport-category')   error   Error from server: Wrong length index getting LOINC language code</p>\n</blockquote>",
        "id": 243617852,
        "sender_full_name": "Eric Haas",
        "timestamp": 1624435047
    },
    {
        "content": "<p><span class=\"user-mention\" data-user-id=\"191316\">@Grahame Grieve</span> this error is still present. see above, don't think this is a caching issue.</p>",
        "id": 243724867,
        "sender_full_name": "Eric Haas",
        "timestamp": 1624490262
    },
    {
        "content": "<p>working on it now</p>",
        "id": 243724958,
        "sender_full_name": "Grahame Grieve",
        "timestamp": 1624490319
    },
    {
        "content": "<p>fixed</p>",
        "id": 243744975,
        "sender_full_name": "Grahame Grieve",
        "timestamp": 1624515031
    }
]